src/ecephys_etl/modules/vbn_create_stimulus_table/create_stim_table.py

```diff
@@ -68,6 +68,13 @@ def get_vsyncs(
 
     falling_edges = sync_dataset.get_falling_edges(vsync_line, units='seconds')
 
+    first_rising_edge = sync_dataset.get_rising_edges(
+        vsync_line, units='seconds'
+    )[0]
+
+    # return only the falling edges that are preceded by rising edges
```

Comment:
Have you confirmed with Corbett that this is the desired behavior?

Reply:
Yes - this is from https://github.com/AllenInstitute/ecephys_etl_pipelines/issues/24